### PR TITLE
T3T-3699 feat: add ParkingPermit and HolidayPeriod entities

### PR DESCRIPTION
**Pull request by simonf7**
As usual not entirely sure its in the right place in the property folder?

### CODE REVIEW

File: src/tabs/apiclient/property/parkingpermit/HolidayPeriod.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    protected $samedateseveryyear;
+
+    // -------------------- Public Functions -------------------- //
+
+    /**
+     * @inheritDoc
+     */
+    public function __construct($id = null)
+    {
+        parent::__construct($id);
```
Comment (alexwyett) [inline src/tabs/apiclient/property/parkingpermit/HolidayPeriod.php:53]: need to set up the date defaults here.

File: src/tabs/apiclient/property/ParkingPermit.php [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    protected $holidayperiods;
+
+    // -------------------- Public Functions -------------------- //
+
+    /**
+     * @inheritDoc
+     */
+    public function __construct($id = null)
+    {
+        parent::__construct($id);
```
Comment (alexwyett) [inline src/tabs/apiclient/property/ParkingPermit.php:90]: Need to initialise the holiday periods collection.

**Review by alexwyett — COMMENTED** [review]
I think property folder is fine, I try to keep them match the api routes as much as possible.

**Comment by simonf7** [comment]
@alexwyett I think (hope) I've done it correctly

**Pull request merged**